src/User/Module.php

```diff
@@ -32,6 +32,10 @@ class Module extends BaseModule
      *               if equals false records will not be deleted
      */
     public $numberSessionHistory = false;
+    /**
+     * @var bool Enable LDAP to sync local users with LDAP users
```

Comment:
Isn't it for search?

Please add documentation for this additional parameter in the `docs` folder as well. Clearly explain the purpose and the requirements for this to work.